client/js/options.js

```diff
@@ -91,6 +93,13 @@ settings.on("change", "input, select, textarea", function() {
 			// otherwise, users get notifications for everything
 			return h !== "";
 		});
+		// Construct regex with wordboundary for every highlight item
+		var highlightsTokens = options.highlights.map(function(h) {
```

Comment:
Sorry to be a pain for nitpicks like this, but could you make this a bit more ES6-like, please?

```js
const highlightsTokens = options.highlights.map((h) => escapeRegExp(h));
```